flang/include/flang/Optimizer/Builder/FIRBuilder.h

```diff
@@ -609,6 +609,17 @@ class FirOpBuilder : public mlir::OpBuilder, public mlir::OpBuilder::Listener {
     return integerOverflowFlags;
   }
 
+  /// Set ComplexDivisionToRuntimeFlag value for whether complex number division
+  /// is lowered to a runtime function by this builder.
```

Comment:
NIT: It's probably not necessary to repeat what the flag does here. If you would prefer to keep it, consider rephrasing

```suggestion
  /// Set ComplexDivisionToRuntimeFlag value. If set to true, complex number
  /// division is lowered to a runtime function by this builder.
```